pyccl/tests/test_baryons.py

```diff
@@ -41,3 +41,33 @@ def test_baryons_from_name():
     bar2 = ccl.Baryons.from_name('Schneider15')
     assert bar.name == bar2.name
     assert bar2.name == 'Schneider15'
+
+
+def test_baryons_in_cosmology():
+    # Test that applying baryons during cosmology creation works.
+    # 1. Outside of cosmo
+    cosmo_nb = ccl.CosmologyVanillaLCDM(
+        transfer_function='bbks', baryonic_effects=None)
+    pk_nb = cosmo_nb.get_nonlin_power()
+    pk_wb = bar.include_baryonic_effects(cosmo_nb, pk_nb)
+    # 2. In cosmo - default BCM model
+    cosmo_wb1 = ccl.CosmologyVanillaLCDM(
+        transfer_function='bbks', baryonic_effects='bcm')
```

Comment:
So this takes the BCM fiducial values then? Can you pass the parameters here too? If so, could you add that to the test?

Reply:
No, you can't pass parameters. The consensus seemed to be that we want to avoid polluting the Cosmology initialization function with so many non-cosmology parameters, so I'd rather do that...

Reply:
Ok